### ISSUE TITLE
Validate IAM path starts and stops with a /

### ISSUE THREAD

**Issue by kddejong** [aws-cloudformation/cfn-lint#3020]
### Describe the feature you'd like to request

Validate IAM path starts and ends with a /

### Describe the solution you'd like

Validate IAM path starts and ends with a /

### Additional context

`The specified value for path is invalid. It must begin and end with / and contain only alphanumeric characters and/or / characters`

Template:
```yaml
RootRole:
    Type: 'AWS::IAM::Role'
    Properties:
      AssumeRolePolicyDocument:
        Version: "2012-10-17"
        Statement:
          - Effect: Allow
            Principal:
              Service:
                - ec2.amazonaws.com
            Action:
              - 'sts:AssumeRole'
      Path: /root
      RoleName: root
      Policies:
        - PolicyName: root
          PolicyDocument:
            Version: "2012-10-17"
            Statement:
              - Effect: Allow
                Action: '*'
                Resource: '*'
```

### Is this something that you'd be interested in working on?

- [X] 👋 I may be able to implement this feature request

### Would this feature include a breaking change?

- [X] ⚠️ This feature might incur a breaking change

**Comment by kddejong** [comment]
This is fixed in v1 using json schema pattern
